src/firebase-db.ts

```diff
@@ -267,6 +268,9 @@ export const watchAllAnswers = (callback: (wrappedAnswer: WrappedDBAnswer[]) =>
   });
 };
 
+// use same universal timezone (UTC) as Lara uses for writing created
+export const createdString = () => (new Date()).toUTCString().replace("GMT", "UTC");
```

Comment:
We are shooting to shutdown the LARA runtime by June. So unless it is going to break something, I'd prefer to use safer values, and more clear names.  I don't think it will break anything, the linked interactive support won't be comparing LARA and AP answers because we don't allow switching like this.

The researcher report code that syncs answers to S3 is looking for the created field: https://github.com/concord-consortium/report-service/tree/master/query-creator#aws-glueathena-setup
So that would have to be updated to include an updated field, however as far as I know none of the reports are using this field yet, so it should just be a schema change.

If we do changes this we will have to deal with this during the migration of student answers from LARA to the AP. We are going to have to migrate most of the built in answers anyway so they are in the right format for their interactive versions. So at that same time we can add the updated value.

But in the interest of getting all of this tested sooner than later, I'm OK with leaving it as is and dealing with it in a follow up PR.